app/Http/Transformers/ContestTransformer.php

```diff
@@ -17,7 +17,11 @@ public function transform(Contest $contest)
     {
         return [
             'id' => $contest->id,
-            // @TODO: More to come...
+            'campaign_id' => $contest->campaign_id,
+            'campaign_run_id' => $contest->campaign_run_id,
+            'signups_open' => $contest->waitingRoom->isOpen(),
+            'signup_start' => $contest->waitingRoom->signup_start_date->toIso8601String(),
+            'signup_end' => $contest->waitingRoom->signup_end_date->toIso8601String(),
```

Comment:
Before when we had a WaitingRoomTransformer, i had it like this, which is more of what I meant to keep a response similar to Phoenix too:

```
        return [
            'id' => $waitingRoom->id,
            'campaign' => [
                'id' => $waitingRoom->campaign_id,
                'campaign_run' => [
                    'id' => $waitingRoom->campaign_run_id,
                ],
            ],
            'signup_dates' => [
                'start' => $waitingRoom->signup_start_date->toISO8601String(),
                'end' => $waitingRoom->signup_end_date->toISO8601String(),
            ],
            'created_at' => $waitingRoom->created_at->toISO8601String(),
            'updated_at' => $waitingRoom->updated_at->toISO8601String(),
        ];
```